flocker/node/agents/blockdevice.py

```diff
@@ -1490,6 +1517,12 @@ def discover_state(self, node_state):
             NodeState(
                 uuid=self.node_uuid,
                 hostname=self.hostname,
+                # FLOC-1874 The sizes of the datasets for these manifestations
+                # will be the size of the actual storage-driver-supplied
+                # volume.  This may not be the size requested in the
+                # configuration.  File a follow-up issue for fixing this if it
+                # matters.  Fixing this requires having access to the
```

Comment:
At minimum, expanding our model to have "this is how much the storage is" and "this is the requested size that we converged on" seems like a good thing.  It will be rather confusing to users when they ask for 7GiB and they get 8GiB and the system doesn't try to converge further.